packages/host/app/services/realm-server.ts

```diff
@@ -82,19 +82,41 @@ export default class RealmServerService extends Service {
       window.localStorage.getItem(sessionLocalStorageKey) ?? undefined;
   }
 
+  async createUser(matrixUserId: string, registrationToken?: string) {
+    await this.login();
+    let response = await this.network.authedFetch(`${this.url.href}_user`, {
```

Comment:
I copied the usage of `this.network.authedFetch`  from `createRealm` from this same service but this feels confusing... why are we providing `Authorization: Bearer ${this.token}` header when `authedFetch` has the  auth functionality to do that for us? 

However `authedFetch` only supports authenticated querying realms, and not the realm server, which is the case here. So given the logic for realm server auth lives in this service (realm-server), I think this should just be a normal `this.network.fetch` instead? @lukemelia @habdelra

Reply:
that seems like a reasonable conclusion